packages/client/ui/react-ui/src/components/dynamic-xyz/DynamicContextProviderWrapper.tsx

```diff
@@ -10,9 +10,12 @@ export interface DynamicContextProviderWrapperProps {
 
 export default function DynamicContextProviderWrapper({
     children,
-    settings,
+    settings = {},
```

Comment:
hm i dont get why these changes are needed? can u explain

Reply:
I reverted these changes, I thought would be cleaner to set defaults but isn't